test/fixtures/bats/comment_style.bats

```diff
@@ -0,0 +1,20 @@
+function should_be_found { # @test
+  true
+}
+
+function should_be_found_with_trailing_whitespace { # @test   
+  true
+}
+
+function should_be_found_with_no_whitespace{#@test
```

Comment:
`shfmt` rejects this syntax FWIW, and it doesn't appear to be valid without spaces before and after the `{`. Should we reject this I wonder?

```bash
$ function should_be_found_with_no_whitespace{ echo 123; }
bash: syntax error near unexpected token `echo'
```

Reply:
Given that bash fails on this, i see 2 paths.  We fail during pre-processing, or simply let it fail when run.  I would suggest the later and simply remove this test.